integration-test/src/main/java/com/scalar/db/transaction/autocommit/AutoCommitTransactionIntegrationTestBase.java

```diff
@@ -0,0 +1,63 @@
+package com.scalar.db.transaction.autocommit;
+
+import com.scalar.db.api.DistributedTransactionIntegrationTestBase;
+import com.scalar.db.config.DatabaseConfig;
+import java.util.Properties;
+import org.junit.jupiter.api.Disabled;
+
+public abstract class AutoCommitTransactionIntegrationTestBase
+    extends DistributedTransactionIntegrationTestBase {
+
+  @Override
+  protected String getTestName() {
+    return "tx_at";
+  }
+
+  @Override
+  protected final Properties getProperties(String testName) {
+    Properties properties = new Properties();
+    properties.putAll(getProps(testName));
+    properties.setProperty(
+        DatabaseConfig.TRANSACTION_MANAGER, AutoCommitTransactionConfig.TRANSACTION_MANAGER_NAME);
+    return properties;
+  }
+
+  protected abstract Properties getProps(String testName);
+
+  @Disabled("auto-commit transaction doesn't support getState()")
+  @Override
+  public void getState_forSuccessfulTransaction_ShouldReturnCommittedState() {}
+
+  @Disabled("auto-commit transaction doesn't support getState()")
+  @Override
+  public void getState_forFailedTransaction_ShouldReturnAbortedState() {}
+
+  @Disabled("auto-commit transaction doesn't support abort()")
+  @Override
+  public void abort_forOngoingTransaction_ShouldAbortCorrectly() {}
+
+  @Disabled("auto-commit transaction doesn't support rollback()")
```

Comment:
Can we change this as follows?:

```suggestion
  @Disabled("Auto-commit transactions don't support rollback()")
```